pkg/reconciler/customrun/customrun_test.go

```diff
@@ -187,7 +188,7 @@ func TestReconcile_CloudEvents(t *testing.T) {
 			}
 
 			ceClient := clients.CloudEvents.(cloudevent.FakeClient)
-			err = eventstest.CheckEventsUnordered(t, ceClient.Events, tc.name, tc.wantCloudEvents)
+			ceClient.CheckCloudEventsUnordered(t, tc.name, tc.wantCloudEvents)
 			if err != nil {
 				t.Errorf(err.Error())
 			}
```

Comment:
There is no `err` returned anymore to check!

Reply:
thanks! I have fixed all of them